apps/fishing-map/features/workspace/vessels/VesselTranslations.ts

```diff
@@ -48,4 +48,11 @@ const translations = 'used_to_get_dynamic_translations_autogenerated'
 // t('vessel.gearTypes.trawler', 'Trawler')
 // t('vessel.gearTypes.purse_seine', 'Purse Seine')
 
+// Pelagic gear types
+// t('vessel.gearTypes.Handline', 'Handline')
```

Comment:
is it ok that one comes with upper case, and the rest not?

Reply:
it's how it was entered in the pelagic dashboard... maybe we could transform everything to lower case when ingesting the data.